### DISCUSSION TITLE
请问钉钉群还有吗？公布那三个都满了，感觉在群里交流可能方便一些

### DISCUSSION THREAD

**Discussion by luoxy1001** [antvis/X6#2159] [Q&A] [answered]
请问钉钉群还有吗？公布那三个都满了，感觉在群里交流可能方便一些

**Comment by NewByVector** [comment, marked as answer]
没有更多群了，后面问题都建议在 Discussions 和 Issues 区讨论，钉钉群消息很难留存，已经解决的问题后面的人还会继续遇到。

**Reply by aiidyc** [reply]
你们不想回答重复的问题，群友看到了会回复帮忙解决问题啊，不能因为问题没有留存连即时交流渠道都关了

**Reply by NewByVector** [reply]
这里 http://x6.antv.antgroup.com/tutorial/about

**Reply by news-ttt** [reply]
> 这里 http://x6.antv.antgroup.com/tutorial/about

您好，咱这个二维码显示失效了

**Reply by NewByVector** [reply]
官网二维码已经更新。
